python_docs_theme/static/pydoctheme.css

```diff
@@ -522,3 +521,9 @@ dl > dt span ~ em {
         overflow-x: auto;
     }
 }
+
+@media (min-width: 1024px) {
```

Comment:
Have you tested this in landscape mode (assuming that's supported)?

Reply:
Here's a narrow landscape mode (iPhone SE):

![image](https://user-images.githubusercontent.com/1324225/218501011-3160eb95-ee70-4f1b-8d3e-c8c9c29a501b.png)

And a wide landscape mode (iPad Air):

![image](https://user-images.githubusercontent.com/1324225/218501262-b896b180-42ac-4105-9a6a-28e3c4ec6a87.png)